### ISSUE TITLE
It returns max 999 rows only

### ISSUE THREAD

**Issue by satishmane** [ghdna/athena-express#1]
Hi,

I used this client, but it returns only 999 rows max. Can you please let me know how do I get more rows

thanks

**Comment by satishmane** [comment]
Thanks for quick response. There is another athena-client package which I guess returns more rows. I mean no limit

**Comment by kvamsij** [comment]
hi,
     when trying query statements like 'show tables' or 'show views' the first result in the list becomes key for every results.Item
     is there a way i can change this in your code?

Thanks,
Vamse.

**Comment by ghdna** [comment]
Let me look into it this a bit. Athena is a bit wacky in structuring the response where there is only 1 column - so for queries like show tables or show views.

**Comment by ghdna** [comment]
> hi,
> when trying query statements like 'show tables' or 'show views' the first result in the list becomes key for every results.Item
> is there a way i can change this in your code?
> 
> Thanks,
> Vamse.

@kvamsij I've just published v3.0 that cleans this up. 
e.g. for Show Tables query
```javascript
{ Items:
   [ { row: 'default' },
     { row: 'sampledb' } ] }
```

**Comment by ghdna** [comment]
> Hi,
> 
> I used this client, but it returns only 999 rows max. Can you please let me know how do I get more rows
> 
> thanks

@satishmane Just published v3.0 that doesn't limit rows to 999. Thanks for bringing it up.
